tests/voip.cpp

```diff
@@ -0,0 +1,431 @@
+#include <gtest/gtest.h>
+
+#include <mtx.hpp>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+using namespace mtx::events;
+
+//! Tests for VoIP v0 support
+TEST(VoIPEvents, CallInviteV0)
```

Comment:
Okay, this is a really minor nitpick, but since you append V0 or V1 to all tests, how about you instead put that as a suffix to the category to match with your comment above it. I.e.

```suggestion
TEST(VoIPEventsV0, CallInvite)
```